src/Application/Duber.Invoice.API/Infrastructure/AutofacModules/MediatorModule.cs

```diff
@@ -1,35 +1,38 @@
-﻿using System.Collections.Generic;
-using System.Reflection;
-using Autofac;
+﻿using Autofac;
 using Duber.Invoice.API.Application.DomainEventHandlers;
 using MediatR;
+using System.Collections.Generic;
+using System.Reflection;
 
 namespace Duber.Invoice.API.Infrastructure.AutofacModules
 {
     public class MediatorModule : Autofac.Module
     {
         protected override void Load(ContainerBuilder builder)
         {
-            builder.RegisterAssemblyTypes(typeof(IMediator).GetTypeInfo().Assembly)
-                .AsImplementedInterfaces();
 
-            // Register all the event classes (they implement IAsyncNotificationHandler) in assembly holding the Commands
+            Autofac.Builder.IRegistrationBuilder<Mediator, Autofac.Builder.ConcreteReflectionActivatorData, Autofac.Builder.SingleRegistrationStyle> registrationBuilder2 = builder.RegisterType<Mediator>().As<IMediator>();
```

Comment:
curious if we could get rid of the variable since you're not using it
```suggestion
builder.RegisterType<Mediator>().As<IMediator>();
```

Reply:
I was in "first do no harm" mode - and just wanted it to build/clear all warnings. That was in the info category and can certainly get rid of it.